source/jormungandr/jormungandr/street_network/geovelo.py

```diff
@@ -0,0 +1,187 @@
+#  Copyright (c) 2001-2017, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+
+from __future__ import absolute_import, print_function, unicode_literals, division
+import logging
+import requests as requests
+import pybreaker
+import json
+from navitiacommon import response_pb2
+from jormungandr import app
+from jormungandr.exceptions import TechnicalError, InvalidArguments
+from jormungandr.street_network.street_network import AbstractStreetNetworkService
+from jormungandr.utils import get_pt_object_coord, is_url, record_external_failure
+
+
+class Geovelo(AbstractStreetNetworkService):
+    sn_system_id = 'geovelo'
```

Comment:
since `sn_system_id` is a common variable for `GeoVelo` `Kraken` and `Valhalla`, i'm wondering would it be better if we define an abstract `get_system_id` in abstract class and implement it in every inheriting class?